### DISCUSSION TITLE
How to set the range of slider?

### DISCUSSION THREAD

**Discussion by BrettKnoss** [fonsp/Pluto.jl#1700] [Questions about using Pluto]
```
@bind x html"<input type=range>"
```
gives a range between 0 and 100, but what If I want a different range like between 1 and 10?

**Comment by disberd** [comment]
You most likely want to use directly Slider from [PlutoUI.jl](https://github.com/JuliaPluto/PlutoUI.jl) which also has a lot of other convenience bounds.
You can also find how it works easily as there is an example PlutoUI notebook among the built-in sample notebooks.

Alternatively, if you want to stick to the custom html implementation without resorting to external packages you can do
```julia
@bind asd html"<input type=range min=0 max=10>"
```
You can check the details of range inputs here: https://www.w3schools.com/tags/att_input_type_range.asp
